### DISCUSSION TITLE
[Feature Request] Sections/chapters

### DISCUSSION THREAD

**Discussion by plaffitt** [asciinema/discussions#102] [Ideas]
It could be interesting for blog posts to be able to define "chapters" with a name in an asciinema record. Chapters could be embedded individually and the whole record would be available too, allowing people to "fast forward" and only watch the full record instead of reading the post if they are in a hurry. It could be useful for long recordings too, to go straight to the interesting part for the viewer.

**Comment by bkil** [comment]
Well, they could always upload the chapters individually under each section of text that describes what is happening. It is usually not optimal for engagement to share videos longer than about a minute anyway.

You could then go the other way, though: concatenate all subparts and offer that up for those who want to watch that instead in full during social events or dinner.

**Comment by ku1ik** [comment]
asciinema player v3.4.0 brings [markers](https://github.com/asciinema/asciinema-player#markers-1), which act as chapter markers, and are displayed as indicators (with label tooltip on hover) on the progress bar.

asciinema.org now has "Markers" field in recording settings form, where markers can be configured.

There's also related new [pauseOnMarkers](https://github.com/asciinema/asciinema-player#pauseonmarkers) option to automatically pause on each enountered marker. If you're using standalone player on your own site then it can be useful in some scenarios.

**Comment by ku1ik** [comment]
The markers can now also be configured on recording settings page, as described here: https://blog.asciinema.org/post/whats-new-in-the-server/
